lms/djangoapps/teams/tests/test_views.py

```diff
@@ -186,7 +202,7 @@ def make_call(self, url, expected_status=200, method='get', data=None, content_t
 
         If a user is specified in kwargs, that user is first logged in.
         """
-        user = kwargs.pop('user', 'student_enrolled')
+        user = kwargs.pop('user', 'student_enrolled_not_on_team')
```

Comment:
@explorerleslie @dianakhuang I did not see any such logic in #9075. Please tag me on the story you create, @explorerleslie. As part of that story, we should think of other places in our code that depend on the "only one team per user" logic. That way, we can consolidate the logic in order to easily remove that requirement down the line.